.github/workflows/testing-framework.yaml

```diff
@@ -2,7 +2,7 @@ name: Testing Framework
 
 on:
   schedule: # schedule the job to run at 12 AM daily
-   - cron: '0 0 * * *'
+   - cron: '0 * * * *'
```

Comment:
I think this means it will run every hour. AFAIK `'0 0 * * *` is once a day.

Reply:
I want this workflow to run once an hour